src/main/java/org/assertj/guava/api/RangeSetAssert.java

```diff
@@ -0,0 +1,982 @@
+    return myself;
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} contains at least one of the given elements.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'> 
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
+   *
+   * rangeSet.add(Range.closed(0, 100));
+   * rangeSet.add(Range.closed(200, 300));
+   * rangeSet.add(Range.closed(500, 600));
+   *
+   * assertThat(rangeSet).containsAnyOf(150, 250, 700);</code></pre>
+   * </p>
+   * 
+   * @param elements the elements to look for in actual {@code RangeSet}.
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} does not contain at least one of the given {@code elements}.
+   * @throws IllegalArgumentException if elements are null or elements are empty while actual is not empty.
+   */
+  public RangeSetAssert<T> containsAnyOf(final T... elements) {
+    rangeSets.assertContainsAnyOf(info, actual, elements);
+    return myself;
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} contains at least one of the given elements.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'> 
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
+   *
+   * rangeSet.add(Range.closed(0, 100));
+   * rangeSet.add(Range.closed(200, 300));
+   * rangeSet.add(Range.closed(500, 600));
+   *
+   * assertThat(rangeSet).containsAnyElementsOf(Arrays.asList(150, 250, 700));</code></pre>
+   * </p>
+   *
+   * @param elements the elements to look for in actual {@code RangeSet}.
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} does not contain at least one of the given {@code elements}.
+   * @throws IllegalArgumentException if elements are null or elements are empty while actual is not empty.
+   */
+  public RangeSetAssert<T> containsAnyElementsOf(final Iterable<T> elements) {
+    rangeSets.assertContainsAnyElementsOf(info, actual, elements);
+    return myself;
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} does not contain the given elements.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'> 
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
+   * 
+   * rangeSet.add(Range.closed(0, 100));
+   * rangeSet.add(Range.closed(200, 300));
+   * rangeSet.add(Range.closed(500, 600));
+   * 
+   * assertThat(rangeSet).doesNotContain(150, 320, 650);</code></pre>
+   * </p>
+   * 
+   * @param elements the elements that should not be present in actual {@code RangeSet}
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} contains the given {@code elements}.
```

Comment:
reword `contains the given ` -> `contains any of the given `

Reply:
fixed